Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -590,6 +590,17 @@ def start_download(self, torrentfilename=None, infohash=None, tdef=None, dconfig
 
         return result
 
+    def get_libtorrent_version(self):
+        """
+        A function to check libtorrent version
```

Comment:
'this method returns the version of the used libtorrent library and is required for compatibility purposes'